packages/deno/src/integrations/deno-cron.ts

```diff
@@ -1,61 +1,56 @@
-import { withMonitor } from '@sentry/core';
-import type { Integration } from '@sentry/types';
-import type { DenoClient } from '../client';
+import { convertIntegrationFnToClass, withMonitor } from '@sentry/core';
+import type { IntegrationFn } from '@sentry/types';
 import { parseScheduleToString } from './deno-cron-format';
 
 type CronOptions = { backoffSchedule?: number[]; signal?: AbortSignal };
 type CronFn = () => void | Promise<void>;
 // Parameters<typeof Deno.cron> doesn't work well with the overloads 🤔
 type CronParams = [string, string | Deno.CronSchedule, CronFn | CronOptions, CronFn | CronOptions | undefined];
 
+const INTEGRATION_NAME = 'DenoCron';
+
+const denoCronIntegration: IntegrationFn = () => {
+  return {
+    name: INTEGRATION_NAME,
+    setupOnce() {
```

Comment:
I moved this to use `setupOnce`, or would we want this to be wrapped multiple times if multiple clients where initialized? 🤔

Reply:
I guess there would be no harm in wrapping it multiple times and it would allow multiple clients 🤔

Reply:
OK, then I'll leave it in setup! 👍